### PR TITLE
Release 0.1

### PR DESCRIPTION
**Pull request by ericfib**
## Patch Notes

- Basic UI
- New page Home
- Show activities home page
- Filter activities home page
- API connection to server

## Some questions
- [x] I understand that my PR won't be merged until GitHub gives a "green light"

### CODE REVIEW

File: app/src/main/java/com/example/workoutapp/ActivityController.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+
+
+public class ActivityController {
+
+    public static final String QUERY_FOR_ACTIVITY = "https://dev-pes-workout.herokuapp.com/api/activity/";
```
Comment (Casassarnau) [inline app/src/main/java/com/example/workoutapp/ActivityController.java:22]: Hauria de ser la de producció https://pes-workout.herokuapp.com/api/activity/

**Review by Casassarnau — APPROVED** [review]
LGTM

**Review by eduardpuga — APPROVED** [review]
Done

**Review by jdconejos — APPROVED** [review]
ok

**Pull request merged**